pages/home.md

```diff
@@ -67,7 +65,7 @@ permalink: /
           <div class="column is-offset-1 is-5">
             <h2 class="title">Store anything</h2>
             <p class="content is-large">
-              Any kind of data can be stored in a Solid pod, including regular files that you might store in a Google Drive or Dropbox folder, but it is the ability to store <b>Linked Data</b> that makes Solid special.
+              Any kind of data can be stored in a Solid pod, including regular files that you might store in a Google Drive or Dropbox folder. What makes Solid special is the ability to store <strong>Linked Data</strong>.
```

Comment:
```suggestion
              Any kind of data can be stored in a Solid pod, including regular documents that you might store in a Google Drive or Dropbox folder. What makes Solid special is the ability to store <strong>Linked Data</strong>.
```

But I'm not a big fan of this line. It's like: well, Drive or Dropbox won't let you store Linked Data, so come to Solid.

But people will be like: I never wanted to store Linked Data‽

Reply:
So ... are you all wedded to the notion of mentioning "Linked Data"  on this page?
Because, I agree with your comment below - that people are more interested in interoperability:

What makes Solid special is the ability to store data in a way that allows interoperability.

Then, we can just make the next section Interoperable data standards (and get away from Linked Data).